examples/01_plotting/plot_3d_map_to_surface_projection.py

```diff
@@ -136,18 +136,35 @@
 # %%
 # Display outlines of the regions of interest on top of a statistical map
 # -----------------------------------------------------------------------
+#
+# Regions can be outlined using both engines.
+# The plotly engine offers more control over the aesthetics of the
+# contours through the lines argument.
 
 figure = plotting.plot_surf_stat_map(fsaverage.infl_right,
                                      texture, hemi='right',
                                      title='Surface right hemisphere',
                                      colorbar=True, threshold=1.,
-                                     bg_map=fsaverage.sulc_right)
+                                     bg_map=fsaverage.sulc_right,
+                                     engine=engine)
+
+if engine == "matplotlib":
+    plotting.plot_surf_contours(fsaverage.infl_right,
+                                parcellation,
+                                labels=labels,
+                                levels=regions_indices,
+                                figure=figure,
+                                legend=True,
+                                colors=['g', 'k'])
+    plotting.show()
+elif engine == "plotly":
+    figure.add_contours(
+        roi_map=parcellation,
+        levels=regions_indices,
+        labels=labels,
+        lines=[{"width": 5}])
+    figure.show()
```

Comment:
I also noticed that the build_docs workflow was stuck because the plotly figure is rendered in a browser window, which needs to be closed for the workflow to move forward.

We could comment this out for the moment. But @Remi-Gau might have a better solution.
```suggestion
    # figure.show()
```

Reply:
Don't have a fix for this yet but there is an issue for it.
https://github.com/nilearn/nilearn/issues/4379

Commenting it out has been my way to deal with it for now.
So you are on track!